metrics_on_dashboards_report/metrics_on_dashboards_report_readme.md

```diff
@@ -0,0 +1,11 @@
+#Metrics on Dashboards Report
+
+This program is a simple console program written in Python 2.7.10, and utilizes the datadogpy API library to return a list of dashboards that have any metric in a series of user provided metrics present in their queries.
+
+This program requires that the datadogpy library be installed on the machine or environment on which the program is running.  Download and installation instructions can be found [here](https://github.com/DataDog/datadogpy)
+
+Once you have copied this program locally, it can be run in a terminal from the directory it is sotred in by running `python metrics_on_dashboards_report.py`.  Simply follow the on screen instructions, and receive a report of Dashboards where any of the metrics you provide are used.
```

Comment:
typo

Reply:
An example of this being run (w/ api/app key redacted) with appropriate input format would be helpful here.